catalog/transformer/catalog-transformer-csv/src/main/java/ddf/catalog/transformer/csv/CsvQueryResponseTransformer.java

```diff
@@ -169,7 +170,7 @@ private void printData(final CSVPrinter csvPrinter, final Iterator iterator) {
   }
 
   private Set<AttributeDescriptor> getAllRequestedAttributes(
-      final List<Result> results, final Set<String> hiddenFields) {
+      final List<Result> results, final Set<String> hiddenFields, HashSet<String> requestedFields) {
```

Comment:
I don't see why we couldn't follow the existing convention here and make that a `final Set<String> requestedFields`